test/ctr.bats

```diff
@@ -614,12 +614,15 @@ function teardown() {
 	[ "$status" -eq 0 ]
 	[[ "$output" == *"$(printf "Stdout:\nhello0 stdout")"* ]]
 
-	stderrconfig=$(cat "$TESTDATA"/container_config.json | python -c 'import json,sys;obj=json.load(sys.stdin);obj["image"]["image"] = "runcom/stderr-test"; json.dump(obj, sys.stdout)')
+	stderrconfig=$(cat "$TESTDATA"/container_config.json | python -c 'import json,sys;obj=json.load(sys.stdin);obj["image"]["image"] = "runcom/stderr-test"; obj["command"] = ["/bin/sleep", "10"]; json.dump(obj, sys.stdout)')
```

Comment:
I would like this to be `infinite` too, but the sleep binary does not support it I think. You have to provide a value and you can specify if you want that in seconds, minutes, hours or days.